superset-frontend/src/dashboard/components/SliceHeaderControls.jsx

```diff
@@ -109,12 +115,14 @@ class SliceHeaderControls extends React.PureComponent {
       updatedDttm,
       componentId,
       addDangerToast,
+      isFullSize,
     } = this.props;
     const cachedWhen = moment.utc(cachedDttm).fromNow();
     const updatedWhen = updatedDttm ? moment.utc(updatedDttm).fromNow() : '';
     const refreshTooltip = isCached
       ? t('Cached %s', cachedWhen)
       : (updatedWhen && t('Fetched %s', updatedWhen)) || '';
+    const resizeLabel = isFullSize ? 'Minimize' : 'Maximize';
```

Comment:
Should use translations

```suggestion
    const resizeLabel = isFullSize ? t('Minimize') : t('Maximize');
```